tensorflow/lite/micro/kernels/transpose_conv.cc

```diff
@@ -105,6 +106,19 @@ TfLiteStatus CalculateOpData(TfLiteContext* context, TfLiteNode* node,
         &data->params.quantized_activation_max,
         data->per_channel_output_multiplier, data->per_channel_output_shift,
         output_channels));
+
+    // Some models have int16 biases which must be converted to int64 for
+    // inference.
```

Comment:
can we have a bug referenced here? It would be good to have additional context for why this is the case and is this a short term workaround or something that we have to live with.

Reply:
Done. Created b/192090531